gui/RespiraConnectedDevice.h

```diff
@@ -0,0 +1,115 @@
+#include "../common/generated_libs/network_protocol/network_protocol.pb.h"
+#include "../common/third_party/nanopb/pb_common.h"
+#include "../common/third_party/nanopb/pb_decode.h"
+#include "../common/third_party/nanopb/pb_encode.h"
+#include "connected_device.h"
+#include <QSerialPort>
+#include <QThread>
+
+#define INTER_FRAME_TIMEOUT 11
```

Comment:
Please use "inline constexpr int INTERFRAME_TIMEOUT_MS" or something (i.e. constant is better than a define; and it's good to include units)